### PR TITLE
Created the Henry Ford page

### PR DESCRIPTION
**Pull request by abel-tefera**
### Summary

- Created the index.html page for displaying Henry Ford's  bio
- Created the index.css file for styling based on the fcc guidelines
- Created an MIT license for this project

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* Styling the body element
```
Comment (abel-tefera) [inline css/style.css]: You should remove all comments in your code. Comments should not end up in the production code.

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// !! IMPORTANT README:
```
Comment (abel-tefera) [inline js/index.js]: Please remove the js file as it's not part of this project

File: README.md [hunk 3]
```diff
@@ -1,2 +1,5 @@
-# past-project-fcc
```
Comment (abel-tefera) [inline README.md:1]: Kindly set up linters in your project

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @abel-tefera, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
- Proper Usage of the Github review feature ✔️ 
- Use Of Gif ✔️
- Use of inline comment ✔️


### Changes Required

- [x] Well done making use of an inline comment in your review, but i would kindly suggest you  include an explanation for `WHY` you request those changes
E.G:

- The student always includes an explanation, a "why", for each requested change. For example:
   - This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
   - This request lacks a "why": Why don't you use an article tag here instead of a div?

- [x] Also i would suggest you make your comment clear and understandable:
- The student comments are clear and easy to understand. For Example:
   - This is a clear comment: *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*
   - This is a vague comment: *The README is not descriptive, why don't you make it better?* 

- [x] [Optional]  I would kindly suggest you make use of friendly `gif` in your in your general review message the way reviewers do use it in their reviews, as this gives your review a kind of friendly attraction to the author of the project you are reviewing.


- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by abel-tefera** [comment]
Hi @abel-tefera 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights


Proper Usage of the Github review feature ✔️
Use Of Gif ✔️
Use of inline comment ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @abel-tefera , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Good use of in-comment
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**